File: lang_py/Gpu.py
from ProcessingUnit import ProcessingUnit

class Gpu(ProcessingUnit):

    def __init__(self, merk, nama, jumlahCore, kecepatanGHz, tipeUnit, vramGB):
        super().__init__(merk, nama, jumlahCore, kecepatanGHz, tipeUnit)
        self.vramGB = vramGB

    def setVramGB(self, vramGB):
        self.vramGB = vramGB

    def getVramGB(self):
        return self.vramGB
    
    

File: lang_java/Gpu.java
class Gpu extends ProcessingUnit{

    private int vramGB;

    public Gpu(){
    }

    public Gpu(String merk, String nama, int jumlahCore, double kecepatanGHz, String tipeUnit, int vramGB){
        super(merk, nama, jumlahCore, kecepatanGHz, tipeUnit);
        this.vramGB = vramGB;
    }

    public void setVramGB(int vramGB){
        this.vramGB = vramGB;
    }

    public int getVramGB(){
        return vramGB;
    }

}